.circleci/config.yml

```diff
@@ -178,6 +177,8 @@ jobs:
                 --build-arg VERSION="${CIRCLE_TAG:-$THISVERSION}" . \
               && e=0 && break || sleep 15
             done && [ "$e" -eq "0" ]
+            # Set wrapper version to match
+            sed -i -E "s/(__version__ = )'[A-Za-z0-9.-]+'/\1'${CIRCLE_TAG:-$THISVERSION}'/" wrapper/fmriprep_docker.py
```

Comment:
I think we can just drop this, since we do it before publishing the wrapper to PyPI
https://github.com/nipreps/fmriprep/blob/ce344b3963caacea367d512787f1a9065932fbc5/.circleci/config.yml#L966